ops-bedrock/devnet-up.sh

```diff
@@ -69,7 +69,7 @@ if [ ! -f ./.devnet/genesis-l1.json ]; then
   echo "Regenerating L1 genesis."
   (
     cd $CONTRACTS_BEDROCK
-    L1_GENESIS_TIMESTAMP=$L1_GENESIS_TIMESTAMP npx hardhat genesis-l1 \
+    L1_GENESIS_TIMESTAMP=$L1_GENESIS_TIMESTAMP npx hardhat --network $NETWORK genesis-l1 \
```

Comment:
I think we should get rid of the env var here and add a flag to the task itself so that we can migrate away from stateful config files that parse env vars

Reply:
done